python/pyarrow/table.pxi

```diff
@@ -1234,8 +1234,8 @@ cdef class Table(_PandasConvertible):
     """
     A collection of top-level named, equal length Arrow arrays.
 
-    Warning
-    -------
+    Warnings
```

Comment:
There is a single warning here, is the plural intended?

Reply:
Yes, it's intended because `numpydoc` does only recognise "Warnings" as a valid header, not "Warning".